### PR TITLE
DP-2707: Adding dashboard docu

### PR DESCRIPTION
**Pull request by Mara3l**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: docs/content/en/docs/administration/permissions/manage_dashboard_permissions.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Provide managing dashboard permissions for user and user groups.
+
+{{% parameters-block  title="Parameters"%}}
+{{< parameter p_name="workspace_id" p_type="str" >}}
+Workspace identification string. e.g. "demo"
+{{< /parameter >}}
+{{< parameter p_name="dashboard_id" p_type="str" >}}
+Dashboard identification string. e.g. "campaign"
+{{< /parameter >}}
+{{< parameter p_name="permissions_for_assignee" p_type="List[CatalogPermissionsForAssignee]" >}}
```
Comment (vojtechtoman) [inline docs/content/en/docs/administration/permissions/manage_dashboard_permissions.md:22]: This does not cover `CatalogPermissionsForAssigneeRule` which can be used to specify an assignee rule (to assign to a set of users)

File: docs/content/en/docs/administration/permissions/list_dashboard_permissions.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+{{< /parameter >}}
+{{< parameter p_name="dashboard_id" p_type="str" >}}
+Dashboard identification string. e.g. "campaign"
+{{< /parameter >}}
+{{% /parameters-block %}}
+
+{{% parameters-block title="Returns" %}}
+
+{{< parameter p_type="CatalogDashboardPermissions" >}}
+Object containing users and user groups and granted dashboard permissions
```
Comment (vojtechtoman) [inline docs/content/en/docs/administration/permissions/list_dashboard_permissions.md:27]: ...and any permission rules in effect

File: docs/content/en/docs/administration/permissions/list_available_assignees.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+weight: 17
+no_list: true
+superheading: "catalog_permission."
+---
+
+
+
+``list_available_assignees(workspace_id: str, dashboard_id: str) -> CatalogAvailableAssignees``
+
+Provide managing dashboard permissions for user and user groups.
```
Comment (vojtechtoman) [inline docs/content/en/docs/administration/permissions/list_available_assignees.md:13]: This one lists users and user groups that can be assigned dashboard permissions for the dashboard in the workspace.

File: docs/content/en/docs/administration/permissions/list_dashboard_permissions.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+weight: 16
+no_list: true
+superheading: "catalog_permission."
+---
+
+
+
+``list_dashboard_permissions(workspace_id: str, dashboard_id: str) -> CatalogDashboardPermissions``
+
+Provide managing dashboard permissions for user and user groups.
```
Comment (vojtechtoman) [inline docs/content/en/docs/administration/permissions/list_dashboard_permissions.md:13]: This one lists the permissions of users and user groups that have access to the dashboard in the workspace, and any permission rules in effect for the dashboard and workspace.

File: docs/content/en/docs/administration/permissions/manage_dashboard_permissions.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+Object containing List of users and user group and desired dashboard permissions. Set empty list permissions for user/user group means remove dashboard permissions.
```
Comment (vojtechtoman) [inline docs/content/en/docs/administration/permissions/manage_dashboard_permissions.md:23]: [...] List of users, user groups, or rules

File: docs/content/en/docs/administration/permissions/manage_dashboard_permissions.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+{{< /parameter >}}
+{{% /parameters-block %}}
+
+{{% parameters-block title="Returns" None="yes" %}}
+{{% /parameters-block %}}
+
+## Example
+
+```python
+
```
Comment (vojtechtoman) [inline docs/content/en/docs/administration/permissions/manage_dashboard_permissions.md:33]:
```
manage_dashboard_permissions(
    workspace_id="demo",
    dashboard_id="sales",
    permissions_for_assignee=[
        CatalogPermissionsForAssignee(
            assignee_identifier=CatalogAssigneeIdentifier(id="demoGroup", type="userGroup"),
            permissions=["EDIT"],
        ),
        CatalogPermissionsForAssigneeRule(
            assignee_rule=CatalogAssigneeRule(type="allWorkspaceUsers"),
            permissions=["VIEW"],
        ),
    ],
)
```

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by hkad98 — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR405

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/219/ : FAILURE in 1m 06s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/369/ : SUCCESS in 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/373/ : SUCCESS in 9s

**Pull request merged**